Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -2672,14 +2684,33 @@ inline std::optional<PositionArea> BuilderConverter::convertPositionArea(Builder
         dim1Axis = PositionAreaAxis::Block;
         dim2Axis = PositionAreaAxis::Inline;
     } else if (!dim1Axis)
-        dim1Axis = positionAreaOppositeAxis(*dim2Axis);
+        dim1Axis = flipPositionAreaAxis(*dim2Axis);
     else if (!dim2Axis)
-        dim2Axis = positionAreaOppositeAxis(*dim1Axis);
+        dim2Axis = flipPositionAreaAxis(*dim1Axis);
 
-    return PositionArea {
+    PositionArea area {
         { *dim1Axis, positionAreaKeywordToTrack(dimPair.first), positionAreaKeywordToSelf(dimPair.first) },
         { *dim2Axis, positionAreaKeywordToTrack(dimPair.second), positionAreaKeywordToSelf(dimPair.second) }
     };
+
+    // Flip according to position-try-fallbacks, if specified.
+    if (const auto& maybePositionTryFallback = builderState.positionTryFallback()) {
+        for (auto tactic : maybePositionTryFallback->tactics) {
+            switch (tactic) {
+            case WebCore::Style::PositionTryFallback::Tactic::FlipBlock:
+                area = flipPositionAreaByLogicalAxis(LogicalBoxAxis::Block, area, builderState.style().writingMode());
+                break;
+            case WebCore::Style::PositionTryFallback::Tactic::FlipInline:
+                area = flipPositionAreaByLogicalAxis(LogicalBoxAxis::Inline, area, builderState.style().writingMode());
+                break;
+            case WebCore::Style::PositionTryFallback::Tactic::FlipStart:
```

Comment:
You can probably drop `WebCore::Style` since you're already in that namespace?

Reply:
Yeah it got added by `clangd` suggestions 😃 I'll remove it

Reply:
Fixed.